### ISSUE TITLE
Read domain from stdin, not from argument

### ISSUE THREAD

**Issue by laerling** [laerling/xkcdget#2]
This is a concern for me because I don't want all the password domains to appear in the history files of my shells.
Instead of supplying the domain as a command line argument I want xkcdget to ask for the domain, e. g.
```
$ xkcdget
domain: github
Master password: 
ThisIsYourPassword_1
```

**Comment by laerling** [comment]
Also, make a PR for pwget, since the concern arises not only for xkcdget.

**Comment by laerling** [comment]
Solved in 6db943fac963ff846ca550f61139baace69367ed and subsequent fix-commit
